README.md

```diff
@@ -124,3 +124,100 @@ completed' button. The unchanged snapshots appear grouped together at the bottom
 
 From here, you can try making your own changes to the app and tests, if you like. If you do, re-run
 the tests  and you'll see any visual changes reflected in Percy.
+
+
+## Playwright with Automate Tutorial
+
+The tutorial assumes you're already familiar with JavaScript and
+[Playwright](https://playwright.dev/docs/library) and focuses on using it with Percy. You'll still
+be able to follow along if you're not familiar with Playwright, but we won't
+spend time introducing Playwright concepts.
+
+The tutorial also assumes you have [Node 14+ with
+npm](https://nodejs.org/en/download/) and
+[git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed.
+
+
+### Step 1
+
+Clone the example application and install dependencies:
+
+```bash
+$ git clone git@github.com:percy/example-percy-playwright.git
+$ cd example-percy-playwright
+$ npm install
+```
+
+### Step 2
+
+Sign in to Percy and create a new project. You can name the project "PercyOnAutomateTest" if you'd like. After
+you've created the project, you'll be shown a token environment variable.
+
+### Step 3
+
+In the shell window you're working in, export the token environment variable:
+
+**Unix**
+
+``` shell
+$ export PERCY_TOKEN="<your token here>"
+```
+
+**Windows**
+
+``` shell
+$ set PERCY_TOKEN="<your token here>"
+
+# PowerShell
+$ $Env:PERCY_TOKEN="<your token here>"
+```
+
+Note: Usually this would only be set up in your CI environment, but to keep things simple we'll
+configure it in your shell so that Percy is enabled in your local environment.
+
+Set the necessary BROWSERSTACK ENVIRONMENT variables
```

Comment:
Can we please add this as well


For automate you will need credentials so refer to following instructions to get the same

1. You will need a BrowserStack `username` and `access key`. To obtain your access credentials, [sign up](https://www.browserstack.com/users/sign_up?utm_campaign=Search-Brand-India&utm_source=google&utm_medium=cpc&utm_content=609922405128&utm_term=browserstack) for a free trial or [purchase a plan](https://www.browserstack.com/pricing).

2. Please get your `username` and `access key` from the [profile](https://www.browserstack.com/accounts/profile) page.

As in other playwright example repos I am seeing this, would be great if you could add this.